kloppy/infra/serializers/event/wyscout/deserializer_v2.py

```diff
@@ -80,6 +80,62 @@ def _generic_qualifiers(raw_event: Dict) -> List[Qualifier]:
     return qualifiers
 
 
+def _create_shot_result_coordinates(raw_event: Dict) -> Optional[Point]:
+    """Estimate the shot end location from the Wyscout tags."""
+    if (
+        _has_tag(raw_event, wyscout_tags.GOAL_LOW_CENTER)
+        or _has_tag(raw_event, wyscout_tags.GOAL_CENTER)
+        or _has_tag(raw_event, wyscout_tags.GOAL_HIGH_CENTER)
+    ):
+        return Point(100.0, 50.0)
+    if (
+        _has_tag(raw_event, wyscout_tags.GOAL_LOW_RIGHT)
+        or _has_tag(raw_event, wyscout_tags.GOAL_CENTER_RIGHT)
+        or _has_tag(raw_event, wyscout_tags.GOAL_HIGH_RIGHT)
+    ):
+        return Point(100.0, 55.0)
+    if (
+        _has_tag(raw_event, wyscout_tags.GOAL_LOW_LEFT)
+        or _has_tag(raw_event, wyscout_tags.GOAL_CENTER_LEFT)
+        or _has_tag(raw_event, wyscout_tags.GOAL_HIGH_LEFT)
+    ):
+        return Point(100.0, 45.0)
+    if _has_tag(raw_event, wyscout_tags.OUT_HIGH_CENTER) or _has_tag(
+        raw_event, wyscout_tags.POST_HIGH_CENTER
+    ):
+        return Point(100.0, 50.0)
+    if (
+        _has_tag(raw_event, wyscout_tags.OUT_LOW_RIGHT)
+        or _has_tag(raw_event, wyscout_tags.OUT_CENTER_RIGHT)
+        or _has_tag(raw_event, wyscout_tags.OUT_HIGH_RIGHT)
+    ):
+        return Point(100.0, 60.0)
+    if (
+        _has_tag(raw_event, wyscout_tags.OUT_LOW_LEFT)
+        or _has_tag(raw_event, wyscout_tags.OUT_CENTER_LEFT)
+        or _has_tag(raw_event, wyscout_tags.OUT_HIGH_LEFT)
+    ):
+        return Point(100.0, 40.0)
+    if (
+        _has_tag(raw_event, wyscout_tags.POST_LOW_LEFT)
+        or _has_tag(raw_event, wyscout_tags.POST_CENTER_LEFT)
+        or _has_tag(raw_event, wyscout_tags.POST_HIGH_LEFT)
+    ):
+        return Point(100.0, 55.38)
```

Comment:
Would it make sense to store the coordinates in constants that can be re-used throughout the deserializer?

For future contributors, it might perhaps not be obvious that 44.62 and 55.38 are the y-coordinates for the goal posts.

Reply:
These coordinates are not used anywhere else, so I don't really see the value of adding more constants. I've added some documentation instead. I hope that's ok too.

Reply:
Thanks. Looks good to me!